src/app/views/grants/container.js

```diff
@@ -37,26 +39,34 @@ var GrantContainerView = Backbone.View.extend({
   handleFilter: function() {
     var filter = this.$el.find('.org-search input').map(this.getData);
     _.each(filter, function(f) {
-      this.filters[f.name] = f.value; 
+      this.filters[f.name] = f.value;
     }.bind(this));
+    this.filters['tag'] = this.$el.find('select.filterbytag').val();
     this.collection.filter(this.filters);
   },
 
   setUpFilter: function() {
     this.$el.find('.org-search input').on('keyup', _.throttle(this.handleFilter.bind(this), 0));
+    this.$el.find('.org-search select.filterbytag').on('change', this.handleFilter.bind(this));
   },
 
-  afterRender: function() {
-    this.setUpFilter();
+  setGrantTagsAndRenderFilters: function(grant_tags) {
```

Comment:
This is quite neatly done. How much slower would it be to pull the `grant_tags` using a method on the collection? I'm a little leery of the parent getting the data from the child, feels out of the typical backbone pattern.